digits/model/images/job.py

```diff
@@ -17,3 +21,16 @@ def __init__(self, **kwargs):
         super(ImageModelJob, self).__init__(**kwargs)
         self.pickver_job_model_image = PICKLE_VERSION
 
+    @override
+    def get_job_stats_as_json_string(self,epoch=-1):
```

Comment:
Hi, I'm still wondering why you didn't just extend `json_dict()` with the additional bits of information?